src/form-generator/form-generator-multi-step.ts

```diff
@@ -182,10 +184,12 @@ export class FormGeneratorMultiStep {
       );
       nextButton.id = classNames.nextButton;
       return nextButton;
-    }
+    } else {
```

Comment:
jezeli masz return w if to nigdy else nie jest potrzebny. 
Ponadto ten sam string sie powtarza w errorze